scipy/sparse/_coo.py

```diff
@@ -813,7 +813,7 @@ class coo_matrix(spmatrix, _coo_base):
 
     Sparse matrices can be used in arithmetic operations: they support
     addition, subtraction, multiplication, division, and matrix power.
-
+    
```

Comment:
Can you take out the 4 white spaces on this line?
Then I think it is ready.
Thanks!

Reply:
Thanks for spotting this, now _coo.py should have no differences compared to the main branch.